client/state/jetpack-connect/reducer.js

```diff
@@ -36,29 +36,39 @@ import {
 	DESERIALIZE
 } from 'state/action-types';
 
+import { isValidStateWithSchema } from 'state/utils';
+import { jetpackConnectSessionsSchema } from './schema';
+
 const defaultAuthorizeState = {
 	queryObject: {},
 	isAuthorizing: false,
 	authorizeSuccess: false,
 	authorizeError: false
 };
 
-function buildNoProtocolUrlObj( url ) {
+function buildNoProtocolUrlObj( url, isInstall ) {
```

Comment:
This will also add `isInstall` to the data that is being saved for `jetpackSSOSessions`, which we shouldn't need in SSO.

Perhaps, instead of adding `isinstall` every time with this function, we could do something like:

```javascript
function buildNoProtocolUrlObj( url, args ) {
	const noProtocolUrl = url.replace( /.*?:\/\//g, '' );
	const noProtocolUrlObj = { [ noProtocolUrl ]: sessionValue };

	return args
		? Object.assign( {}, args, noProtocolUrlObj )
		: noProtocolUrlObj
}
```